config.json

```diff
@@ -1,6 +1,7 @@
 {
   "slug": "rust",
   "language": "Rust",
+  "ignore_pattern": "example|description.md|metadata.yml",
```

Comment:
another fun fact: the ignore_pattern is not anchored so you'll also match `the_description.md_file_contains_the_description`